tests/scanner/audit/bigquery_rules_engine_test.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests the BigqueryRulesEngine."""
+
+import copy
+import itertools
+import mock
+import yaml
+
+from tests.unittest_utils import ForsetiTestCase
+from google.cloud.security.common.util import file_loader
+from google.cloud.security.scanner.audit.errors import InvalidRulesSchemaError
+from google.cloud.security.scanner.audit import base_rules_engine as bre
+from google.cloud.security.scanner.audit import bigquery_rules_engine as bqe
+from google.cloud.security.scanner.audit import rules as scanner_rules
+from tests.unittest_utils import get_datafile_path
+
+
+# TODO: Define more tests
```

Comment:
TODO's should be more specific about what is waiting to be done.  So, that the another person who sees this will know what to do.  I think that some basic testings for ACL violations (email, domain, etc) as well as for ACL with no violations, should be noted.  These are really important tests, so it would be super-awesome if they are included in this PR.